main.ts

```diff
@@ -20,34 +26,84 @@ const DEFAULT_SETTINGS: PluginSettings = {
   lastUpdated: null,
   autoSync: true,
   highlightSortOldestToNewest: true,
+  syncNotesOnly: false,
   logFile: true,
   logFileName: 'Sync.md',
+  frontMatter: false,
+  frontMatterTemplate: `---
+id: {{ id }}
+updated: {{ updated }}
+title: {{ title }}
+author: {{ author }}
+---
+`,
+  headerTemplate: `
+%%
+ID: {{ id }}
+Updated: {{ updated }}
+%%
+
+![]( {{ cover_image_url }})
+
+# About
+Title: {{ title }}
+Authors: {{ authorStr }}
+Category: #{{ category }}
+{%- if tags %}
+Tags: {{ tags }}
+{%- endif %}
+Number of Highlights: =={{ num_highlights }}==
+Readwise URL: {{ highlights_url }}
+Source URL: {{ source_url }}
+Date: [[{{ updated }}]]
+Last Highlighted: *{{ last_highlight_at }}*

```

Comment:
Please add an empty line between `Last Highlighted` and `---`. Otherwise, Obsidian's markdown viewer will enlarge the text above the `---` line while in edit mode:

![about2](https://user-images.githubusercontent.com/4033621/149646063-c86466d3-034b-4585-80c4-b8b2618c37e9.png)